.github/workflows/security.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: security
+
+on:
+  push:
+    branches: [main]
+  release:
+    types: [published]
+  pull_request:
+    branches: [main]
```

Comment:
same about `main` branch

Reply:
added in fixup commit for the specific changes: https://github.com/scality/osis/pull/143/commits/7050fcb1546cee061594fdc81b7fc9bbbbe7d161

Reply:
Updated link: https://github.com/scality/osis/pull/143/commits/2c118c55d4631a63a2ed87d9986b25a18c0f4c93